render/render.go

```diff
@@ -5,10 +5,10 @@ import (
 )
 
 // MapFunc is anything which can take an arbitrary Node and
-// return a set of other Nodes.
+// return another Node.
 //
-// If the output is empty, the node shall be omitted from the rendered topology.
-type MapFunc func(report.Node) report.Nodes
+// If the output ID is blank, the node shall be omitted from the rendered topology.
+type MapFunc func(report.Node) report.Node
```

Comment:
The more idiomatic approach would be to return `(report.Node, bool)`, but I do dislike the clutter that introduces. Perhaps add a comment to that effect.